core/src/main/java/org/elasticsearch/search/rescore/QueryRescorerBuilder.java

```diff
@@ -78,6 +75,25 @@ public QueryRescorerBuilder(QueryBuilder<?> builder) {
     }
 
     /**
+     * Read from a stream.
+     */
+    public QueryRescorerBuilder(StreamInput in) throws IOException {
+        super(in);
```

Comment:
nitpick: super(in) will be called anyways right? it reads odd that doWriteTo doesn't call super but the constructor does. The whole idea behind doWriteTo was exactly not to forget calling super in fact :)